.github/workflows/docker-build-push.yaml

```
@@ -0,0 +1,369 @@
+            if [ -z "$package_name" ]
+            then
+              package_name="${{ github.event.repository.name }}"
+            fi
+            echo "package_name=$package_name" | tee -a "$GITHUB_OUTPUT"
+
+            echo "enabled=true" | tee -a "$GITHUB_OUTPUT"
+          fi
+
+      - name: 'Amazon Elastic Container Registry: Configure AWS credentials'
+        if: steps.amazon.outputs.enabled == 'true'
+        uses: aws-actions/configure-aws-credentials@v5
```

actionlint (1.7.8)

missing input "aws-region" which is required by action "aws-actions/configure-aws-credentials@v5". all required inputs are "aws-region"

(action)